ft_crossfrequencyanalysis.m

```diff
@@ -82,40 +93,46 @@
 freqlow  = ft_checkdata(freqlow,  'datatype', 'freq', 'feedback', 'yes');
 freqhigh = ft_checkdata(freqhigh, 'datatype', 'freq', 'feedback', 'yes');
 
-% prior to 19 Jan 2017 this function had input options cfg.chanlow and cfg.chanhigh,
-% but nevertheless did not support between-channel CFC computations
-cfg = ft_checkconfig(cfg, 'forbidden', {'chanlow', 'chanhigh'});
+if isfield(cfg, 'chanlow') && isfield(cfg, 'chanhigh')
+    docrosschan = true;
+    cfg.chanlow  = ft_channelselection(cfg.chanlow, freqlow.label);
+    cfg.chanhigh  = ft_channelselection(cfg.chanhigh, freqhigh.label);
+    labelcmb = ft_channelcombination({cfg.chanlow,cfg.chanhigh},union(freqlow.label, freqhigh.label));
```

Comment:
This is not robust. It assumes that chanlow and chanhigh are cell-arrays with the same dimensionality (number of elements). This is not in general guaranteed. It seems that the desired behavior is to have an all-by-all pairwise low-to-high CFC, i.e. an NxM matrix of CFC estimates. I don't see a need to kick out the auto combinations a priori, nor to represent the data in a linear array of 'labelcmb'

Reply:
I do not have any such assumption. The pairwise low-to-high CFC would be a (N*M) x 2 matrix, where each row describes a channel pair from where the CFC is computed. In this way, the data has the same dimensionality for both within- and cross-channel analysis. with dimord `chan_freqlow_freqhigh` (for within) and `chancmb_freqlow_freqhigh` (for cross).